launch_ros/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model
+  href="http://download.ros.org/schema/package_format2.xsd"
+  schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>launch_ros</name>
+  <version>0.4.0</version>
+  <description>ROS specific extensions to the launch tool.</description>
+  <maintainer email="william@osrfoundation.org">William Woodall</maintainer>
+  <license>Apache License 2.0</license>
+
+  <depend>launch</depend>
```

Comment:
this also depends on rclpy, lifecycle_msgs, ament_index_python and osrf_pycommon

Reply:
Yes, but I don't think `osrf_pycommon` directly. cd0ebe2

Reply:
You were right, I was wrong, here's the fixup: 15ac0c3